packages/flutter/lib/src/widgets/container.dart

```diff
@@ -52,6 +52,7 @@ import 'image.dart';
 ///  * [Decoration], which you can extend to provide other effects with
 ///    [DecoratedBox].
 ///  * [CustomPaint], another way to draw custom effects from the widget layer.
+///  * [SliverDecoration], which paints a [Decoration] around a sliver.
```

Comment:
Another of these bullet points says BoxDecoration and this says Decoration, should they be the same or is there a difference here?